### PR TITLE
Added condition to resolve issue in Rhel7.6alt

### PR DESCRIPTION
**Pull request by shirishaganta**
Based on issue #1406 modified the script to run correctly

Signed-off-by: Shirisha Ganta <shiganta@in.ibm.com>

### CODE REVIEW

**Comment by shirishaganta** [comment]
Before Patch:
JOB ID : 15083accedf6804440053110f3c74fae5fe31937
JOB LOG : /home/avocado-fvt-wrapper-py2/results/job-2019-11-14T11.10-15083ac/job.log
(1/3) perf_uprobe.py:PerfUprobe.test_uprobe: PASS (8.03 s)
(2/3) perf_uprobe.py:PerfUprobe.test_uprobe_return: ERROR: Command 'perf record -o /var/tmp/tmpY1XkKw -e probe_uprobe_test:doit__return -- ./uprobe_test' failed.\nstdout: ''\nstderr: "event syntax error: 'probe_uprobe_test:doit__return'\n \___ unknown tracepoint\n\nError:\tFile /sys/kernel/debu... (0.84 s)
(3/3) perf_uprobe.py:PerfUprobe.test_uprobe_variable: PASS (1.04 s)
RESULTS : PASS 2 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME : 10.46 s
JOB HTML : /home/avocado-fvt-wrapper-py2/results/job-2019-11-14T11.10-15083ac/results.html

After Patch:
JOB ID : 9521872a2ae64905ef5d0c110a8ee1b7afa92aa8
JOB LOG : /home/avocado-fvt-wrapper-py2/results/job-2019-11-14T15.37-9521872/job.log
(1/3) perf_uprobe.py:PerfUprobe.test_uprobe: PASS (2.72 s)
(2/3) perf_uprobe.py:PerfUprobe.test_uprobe_return: PASS (1.13 s)
(3/3) perf_uprobe.py:PerfUprobe.test_uprobe_variable: PASS (1.05 s)
RESULTS : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME : 5.46 s
JOB HTML : /home/avocado-fvt-wrapper-py2/results/job-2019-11-14T15.37-9521872/results.html

File: perf/perf_uprobe.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def setUp(self):
         smm = SoftwareManager()
         detected_distro = distro.detect()
         self.distro_name = detected_distro.name
+        self.distro_version = detected_distro.version
```
Comment (PraveenPenguin) [inline perf/perf_uprobe.py]: why global param assignemt

File: perf/perf_uprobe.py [hunk 2]
```diff
@@ -94,7 +95,7 @@ def test_uprobe_return(self):
         if 'Added new event' not in output.stderr.decode("utf-8"):
             self.fail("perf: probe of uprobe return failed")
         # RHEL
-        if self.distro_name == "rhel":
+        if self.distro_name == "rhel" and self.distro_version > "7":
```
Comment (PraveenPenguin) [inline perf/perf_uprobe.py]: it can be just like `if self.distro_name == "rhel" and detected_distro.distro_version > "7"`

**Comment by shirishaganta** [comment]
As you requested above, I changed the script
After Patch:
JOB ID     : 3127a01e7cbd86c98970f4b9794be4b3f25d46c2
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2019-12-11T11.41-3127a01/job.log
 (1/3) perf_uprobe.py:PerfUprobe.test_uprobe: PASS (3.98 s)
 (2/3) perf_uprobe.py:PerfUprobe.test_uprobe_return: PASS (2.09 s)
 (3/3) perf_uprobe.py:PerfUprobe.test_uprobe_variable: PASS (1.98 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.74 s
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2019-12-11T11.41-3127a01/results.html

**Comment by harish-24** [comment]
@shirishaganta Does the above condition hold good for Rhel 7.7 and 7.8 as well?

**Comment by shirishaganta** [comment]
> @shirishaganta Does the above condition hold good for Rhel 7.7 and 7.8 as well?

It holds good for both rhel 7.7 and rhel 7.8

**Comment by PraveenPenguin** [comment]
@harish-24 please ACK it if it looks good

**Pull request merged**